omoefe-dukuye/SendIT
# chore(api): Refactor code to respect SRP

#### What this PR does
This Pull Request, if merged,  will refactor the code to respect the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single_responsibility_principle) (SRP). Makes for easier maintainability.
(Starts [#161832653](https://www.pivotaltracker.com/story/show/161832653))

#### Description of Tasks
- Install router middleware
- Refactor callback functions
- Improve test coverage

#### How to test manually
- Clone the repo `git clone https://github.com/un-tethered/SendIT.git`
- Checkout to the branch `git checkout ch-refactor-161832653`
Run `npm install` to install application dependencies
Browse through code to see better arranged modules. 

#### Pivotal tracker story
[#161832653](https://www.pivotaltracker.com/story/show/161832653)
